litestar/contrib/sqlalchemy/base.py

```diff
@@ -127,14 +128,19 @@ def to_dict(self, exclude: set[str] | None = None) -> dict[str, Any]:
         return {field.name: getattr(self, field.name) for field in self.__table__.columns if field.name not in exclude}
 
 
-meta = MetaData(naming_convention=convention)
-orm_registry = registry(
-    metadata=meta,
-    type_annotation_map={UUID: GUID, EmailStr: String, AnyUrl: String, AnyHttpUrl: String, dict: JSON},
-)
+def create_registry() -> registry:
+    """Create a new SQLAlchemy registry."""
+    meta = MetaData(naming_convention=convention)
+    return registry(
+        metadata=meta,
+        type_annotation_map={UUID: GUID, EmailStr: String, AnyUrl: String, AnyHttpUrl: String, dict: JSON},
+    )
+
+
+orm_registry = create_registry()
```

Comment:
@cofin - this alright? Made it easier for me to reproduce isolated metadata instances for docs examples.